### ISSUE TITLE
Merge participant and user model

### ISSUE THREAD

**Issue by OtisRed** [CodeForPoznan/codeforpoznan.pl_v3#390]
#### Subject of the issue:
Given that we consider possibility for creating individual accounts it makes sense to merge User and Participant model into one entity. (I guess even if we weren't going for individual account it would still make sense)

#### Actual behaviour:
Currently we have to object models:
1. User (fields: id, username, password)
2. Participant (fields: id, first_name, last_name, email, github, phone, slack)

#### Expected behaviour:
We have only one model "User" that includes:
* id 
* first_name 
* last_name 
* email 
* github 
* phone
* slack
* password
* isAdmin (true/false)

#### Notes
Migration based on that change will be solved in a separate task - don't merge before it's ready

**Comment by w1stler** [comment]
Couple of questions:

1. What about current users and participants?
2. What is that migration referenced in notes?
3. We should take into consideration migrating list of presences into new schema (the one from csv is not up to date with what we have beed doing recently, right)?

**Comment by OtisRed** [comment]
1. The general idea is to make all the participant `users` according to the schema desribed 
2. This will require migrating current `participants` into `users` - which is what the note refers to. After discussion with @arturtamborski we thought that dividing the change of the model and the subsequent migration of current participants into that model will be easier to review - assuming that merging will be handled carefully in order. 
3. Yes. Migration task would actually require performing three migrations: migrating participants into users, migrating attendance lists changing references from participants to newly created users and finally deleting partitipants 

This was the solution we thought of. We could alays discuss this, of course.

**Comment by w1stler** [comment]
It's all fine, I can take care of all of these, just wanted to clarify the order. :)
